ansible/roles/ceph-grafana/defaults/main.yml

```diff
@@ -21,6 +21,11 @@ carbon:
   storage_dir:
     yum: /var/lib/carbon
     apt: /var/lib/graphite
+whisper:
+  retention:
+    - ['10s', '7d']
```

Comment:
A comment describing the format of this data would go a long way e.g. # 10s interval retention for the first 7 days worth of data. What do you think?